packages/contracts/scripts/ovm-gas-oracle.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+set -e
+
+RPC_URL=${RPC_URL:-http://localhost:8545}
+OVM_GAS_ORACLE=0x420000000000000000000000000000000000000F
+
+function send_tx() {
+    cast send --rpc-url $RPC_URL \
+        --private-key $PRIVATE_KEY \
+        --legacy \
+        --gas-price 0 \
+        $OVM_GAS_ORACLE \
+        $1 \
+        $2
+}
+
+function call() {
+    cast call --rpc-url $RPC_URL \
+        $OVM_GAS_ORACLE \
+        $1
+}
+
+echo "Scalar:       $(call 'scalar()(uint256)')"
+echo "L2 gas price: $(call 'gasPrice()(uint256)')"
+echo "Overhead:     $(call 'overhead()(uint256)')"
+
+if [[ ! -z $PRIVATE_KEY ]]; then
```

Comment:
nit: maybe this should be mandatory? Otherwise, if it's not set, the user may not notice the script doing nothing as it exits zero. This isn't blocking though.

Reply:
Without the key, it will just print the values - I did that intentionally, but if that is not desired I can update the script

Reply:
(resolving this so we can merge the PR)

Reply:
ah ok, that's totally fine since it's intentional.